### PR TITLE
To-Do List (Add & Remove) 

### PR DESCRIPTION
**Pull request by aliatayee**
## To-Do List (Add & Remove) 
in this milestone i have developed the following parts:
- Add task functionality through user input and store to local storage
- Get task list from local storage
- Remove functionality when click on remove button
- Edit functionality by clicking on tasks
- Completed task functionality

### CODE REVIEW

File: src/modules/functionalities.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+};
+
+const editTodo = (index, val, todos) => {
+  todos.forEach((item) => {
+    if (item.index == index) {
+      item.description = val;
+    }
+  });
+  addToLocalStorage(todos);
+};
```
Comment (thecodechaser) [inline src/modules/functionalities.js:36]: - [ ] When I edit the task description the app is duplicating the index id, Please make sure to fix this issue.

### See the image for reference 

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @aliatayee👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/functionalities.js (hunk 1 above)
Comment (aliatayee) [reply src/modules/functionalities.js:36]: @thecodechaser  thank you but for me it is working fine. can you please clear storage and try once again?

Comment (thecodechaser) [reply src/modules/functionalities.js:36]: @aliatayee Hii, I tried 4-5 times I faced the same issue every time.

**Review by roshan-bajgain — APPROVED** [review]
Hi @aliatayee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**